### ISSUE TITLE
ClusterRoleBinding is deprecated warning when importing a k3s 1.19.3 cluster

### ISSUE THREAD

**Issue by dnoland1** [rancher/rancher#30043]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
1. Install a k3s 1.19.3 cluster.
2. On a Rancher v2.5.2 create an imported cluster.
3. Run the kubectl command to import the k3s cluster into Rancher.

**Result:**
```
# kubectl apply -f https://myserver/v3/import/longstringblahblahblahhereredacted.yaml
clusterrole.rbac.authorization.k8s.io/proxy-clusterrole-kubeapiserver created
clusterrolebinding.rbac.authorization.k8s.io/proxy-role-binding-kubernetes-master created
namespace/cattle-system created
serviceaccount/cattle created
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding
clusterrolebinding.rbac.authorization.k8s.io/cattle-admin-binding created
secret/cattle-credentials-ebf24cc created
clusterrole.rbac.authorization.k8s.io/cattle-admin created
deployment.apps/cattle-cluster-agent created
```

**Other details that may be helpful:**
This warning mentioned above should be fixed:
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
v2.5.2
- Installation option (single install/HA):
HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
Imported
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
aws cloud 2CPU, 4GB RAM
- Kubernetes version (use `kubectl version`):
```
# kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3+k3s2", GitCommit:"f8a4547bec36b4d1597c19c86b5063756d4c150e", GitTreeState:"clean", BuildDate:"2020-10-30T19:30:45Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3+k3s2", GitCommit:"f8a4547bec36b4d1597c19c86b5063756d4c150e", GitTreeState:"clean", BuildDate:"2020-10-30T19:30:45Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):
NA, containerd

gz#13611

**Comment by innobead** [comment]
@cclhsu please help with this.

**Comment by deniseschannon** [comment]
@nickgerace Can you help make a backport PR to release/v2.5 for this?

**Comment by kravciak** [comment]
I was able to reproduce in `v2.5.2` and verified it's fixed in `v2.5-head 03b8153` and `master-head 98fe72a`
I tested with
 - ubuntu 20.04 and SLES-15.2
 - k3s v1.20.2+k3s1 and v1.19.3+k3s3
